src/getPullRequest.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const fetch = require('node-fetch')
```

Comment:
use octokit which is already available to you https://github.com/actions/toolkit/tree/main/packages/github

Reply:
Very cool! I did not realize I had this tool at my disposal.